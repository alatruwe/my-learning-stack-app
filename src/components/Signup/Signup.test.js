import React from "react";
import toJson from "enzyme-to-json";
import Signup from "./Signup";
import { shallow } from "enzyme";

describe(`Signup component`, () => {
  it("renders the component by default", () => {
    const wrapper = shallow(<Signup />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
