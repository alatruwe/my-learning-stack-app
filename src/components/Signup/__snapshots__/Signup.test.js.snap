// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup component renders the component by default 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
